userstatus/userstatuswwu/classes/userstatuswwu.php

```diff
@@ -180,33 +180,29 @@ private function get_all_ziv_users() {
                 if (strpos($currentstring['0'], '@')) {
                     continue;
                 }
+                $currentgroup = rtrim($currentstring[1]);
                 // In case no groups were determined the default is used.
-                if (empty($this->groups) && count($this->groups) == null) {
-                    // Additional check whether there is a second word in the current line.
-                    if (array_key_exists(1, $currentstring)) {
```

Comment:
Warum hast du diesen Check entfernt? Der überprüft ob eine Zeile zwei Wörter enthält, ein Wort Zeilen können wohl auftreten und sind irrelevant für uns :)

Reply:
In Zeilen 175-178 wird schon sichergestellt, dass die Zeile zwei Wörter hat, da ist der Check hier doppelte Arbeit :)

Reply:
true never mind...